### DISCUSSION TITLE
Option to hide Factorissimo surfaces from Space Map

### DISCUSSION THREAD

**Discussion by nicholasgower** [notnotmelon/factorissimo-2-notnotmelon#110] [Ideas]
I'm working on a mod that generates six clones of each planet in Space Age to represent temperate and polar zones of each planet. This naturally makes the space map more cluttered, and combined with Factorissimo, the clutter becomes unbearable. If there was an option to hide all of the Factorissimo surfaces from the Space map, I would appreciate that.
![image](https://github.com/user-attachments/assets/052857b3-62b5-4389-9d85-70d9b493960f)

**Comment by notnotmelon** [comment]
Is it possible in the API? I don't think so
